internal/mode/static/nginx/modules/src/httpmatches.js

```diff
@@ -41,8 +41,7 @@ function extractMatchesFromRequest(r, matches) {
 		verifyMatchList(matchList);
 	} catch (e) {
 		throw Error(
-			`cannot redirect the request; ${matchList} matches list is not valid`,
-			e.message,
+			`cannot redirect the request; error parsing ${r.variables[MATCHES_KEY]} into a JSON object: ${e}`,
```

Comment:
the JSON parsing is no longer relevant here.
Let's have something like this (what was before + the error message from verifyMatchList exception):
```
cannot redirect the request; ${matchList} matches list is not valid: ${e}
```